src/main/resources/application.yaml

```diff
@@ -15,6 +15,52 @@ management:
 spring:
   application:
     name: Spring Boot Template
+
+security:
+  anonymousPaths:
+    - "/"
+    - "/health"
+    - "/health/liveness"
+    - "/actuator/**"
+    - "/loggers/**"
+    - "/swagger-ui.html"
+    - "/swagger-resources/**"
+    - "/v2/api-docs"
+    - "/webjars/springfox-swagger-ui/**"
+    - "/search/**"
+    - "/csrf"
+    - "/error"
+    - "/favicon.ico"
+  ### Update list with service id of any service going to interact with casse document api
+  authorisedServices:
+    - ccd-case-document-am-api
+    - ccd_gw
+    - xui_webapp
+
+
+
+### dependency configuration
+auth.provider.service.client.baseUrl: ${IDAM_S2S_URL:http://localhost:4502}
+idam:
+  s2s-auth:
+    totp_secret: ${S2S_SECRET:AAAAAAAAAAAAAAAB}
+    microservice: ccd-case-document-am-api
+    url: ${S2S_URL:http://localhost:4502}
+    pbaFromUserAccountMap: false
+usermap:
+  pbaFromUserAccountMap:
+    - "false"
+
+
+auth.idam.client.baseUrl: ${idam_url:http://localhost:4501}
+
+
+
+#auth.idam.client.detailsUri: "/o/userinfo"
+## Secure roles configuration
```

Comment:
What are the variable name which can be configured to hold secure roles? can you please add that and maybe comment it.

Reply:
but this placeholder cannot work if we can't map it some configuration properties which feeds in some class variables. so we are definitely missing the property name here.

Reply:
I have added the placeholder authorisedRoles.